### DISCUSSION TITLE
%E5%8D%9A%E5%AE%A2/%E8%B5%A0%E8%A8%80

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#456] [Announcements]
# %E5%8D%9A%E5%AE%A2/%E8%B5%A0%E8%A8%80

且视他人之疑目如盏盏鬼火，大胆地去走自己的夜路。 —— 史铁生《病隙碎笔》

http://localhost:8080/%E5%8D%9A%E5%AE%A2/%E8%B5%A0%E8%A8%80.html

**Comment by Scrates1** [comment]
史铁生呀。
